<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-11-01T12:19:23 WAT" name="Default suite" finished-at="2021-11-01T12:19:33 WAT" duration-ms="10048">
    <groups>
    </groups>
    <test started-at="2021-11-01T12:19:23 WAT" name="Default test" finished-at="2021-11-01T12:19:33 WAT" duration-ms="10048">
      <class name="rendition.YmlTester">
        <test-method is-config="true" signature="initialization()[pri:0, instance:rendition.YmlTester@63a65a25]" started-at="2021-11-01T12:19:21 WAT" name="initialization" finished-at="2021-11-01T12:19:23 WAT" duration-ms="2136" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method signature="test()[pri:0, instance:rendition.YmlTester@63a65a25]" started-at="2021-11-01T12:19:23 WAT" name="test" finished-at="2021-11-01T12:19:33 WAT" duration-ms="10007" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "java.util.Map.get(Object)" because "rendition.YmlTester.Property" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "java.util.Map.get(Object)" because "rendition.YmlTester.Property" is null
at rendition.YmlTester.getLocator(YmlTester.java:40)
at rendition.YmlTester.Click(YmlTester.java:47)
at rendition.YmlTester.test(YmlTester.java:55)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method is-config="true" signature="beforeMethod(java.lang.reflect.Method, org.testng.ITestResult)[pri:0, instance:rendition.YmlTester@63a65a25]" started-at="2021-11-01T12:19:33 WAT" name="beforeMethod" finished-at="2021-11-01T12:19:33 WAT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void rendition.YmlTester.test() throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[TestResult name=test status=FAILURE method=YmlTester.test()[pri:0, instance:rendition.YmlTester@63a65a25] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method is-config="true" signature="closeDriverObjects()[pri:0, instance:rendition.YmlTester@63a65a25]" started-at="2021-11-01T12:19:33 WAT" name="closeDriverObjects" finished-at="2021-11-01T12:19:38 WAT" duration-ms="4689" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDriverObjects -->
      </class> <!-- rendition.YmlTester -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
